src/ring_theory/localization.lean

```diff
@@ -266,6 +266,16 @@ lemma mk'_spec' (x) (y : M) :
   algebra_map R S y * mk' S x y = algebra_map R S x :=
 (to_localization_map M S).mk'_spec' _ _
 
+@[simp]
+lemma mk'_spec_mk (x) (y : R) (hy : y ∈ M) :
+  mk' S x ⟨y, hy⟩ * algebra_map R S y = algebra_map R S x :=
+mk'_spec S x ⟨y, hy⟩
+
+@[simp]
+lemma mk'_spec'_mk (x) (y : R) (hy : y ∈ M) :
```

Comment:
This name looks a bit funny. But I guess you don't want to put the prime at the end, so you put it after `spec`?

Reply:
Indeed, both because ending with `_mk'` would suggest `is_localization.mk'` and because this is the `subtype.mk` version of `mk'_spec'` defined on line 265.